--- CGSInternal/CGSConnection.h.orig	2011-07-18 12:18:33.000000000 +0200
+++ CGSInternal/CGSConnection.h	2011-07-18 12:21:00.000000000 +0200
@@ -29,10 +29,6 @@
 
 CG_EXTERN_C_BEGIN
 
-/*! DOCUMENTATION PENDING - verify this is Leopard only! */
-CG_EXTERN CGError CGSSetLoginwindowConnection(CGSConnectionID cid) AVAILABLE_MAC_OS_X_VERSION_10_5_AND_LATER;
-CG_EXTERN CGError CGSSetLoginwindowConnectionWithOptions(CGSConnectionID cid, CFDictionaryRef options) AVAILABLE_MAC_OS_X_VERSION_10_5_AND_LATER;
-
 /*! Enables or disables updates on a connection. The WindowServer will forcibly reenable updates after 1 second. */
 CG_EXTERN CGError CGSDisableUpdate(CGSConnectionID cid);
 CG_EXTERN CGError CGSReenableUpdate(CGSConnectionID cid);
--- CGSInternal/CGSWindow.h.orig	2011-07-18 12:19:27.000000000 +0200
+++ CGSInternal/CGSWindow.h	2011-07-18 12:21:31.000000000 +0200
@@ -169,9 +169,6 @@
 /*! DOCUMENTATION PENDING */
 CG_EXTERN CGError CGSWindowAnimationChangeLevel(CGSAnimationObj animation, CGWindowLevel level);
 
-/*! DOCUMENTATION PENDING */
-CG_EXTERN CGError CGSWindowAnimationSetParent(CGSAnimationObj animation, CGSWindowID parent) AVAILABLE_MAC_OS_X_VERSION_10_5_AND_LATER;
-
 /*! Releases a window animation. */
 CG_EXTERN CGError CGSReleaseWindowAnimation(CGSAnimationObj animation);
 
--- CGSInternal/CarbonHelpers.h.orig	2011-07-18 12:19:39.000000000 +0200
+++ CGSInternal/CarbonHelpers.h	2011-07-18 12:20:30.000000000 +0200
@@ -25,16 +25,3 @@
 
 #pragma once
 #include "CGSWindow.h"
-
-
-CG_EXTERN_C_BEGIN
-
-/* Gets a CGSWindowID for a WindowRef. Implemented in Carbon.framework. 
-   This call is deprecated in 10.5. Please use the public alternative, `HIWindowGetCGWindowID`. */
-CG_EXTERN CGSWindowID GetNativeWindowFromWindowRef(WindowRef ref) DEPRECATED_IN_MAC_OS_X_VERSION_10_5_AND_LATER;
-
-/* Gets a WindowRef (in the current process) from a CGSWindowID. Implemented in Carbon.framework.
-   This call is deprecated in 10.5. Please use the public alternative, `HIWindowFromCGWindowID`. */
-CG_EXTERN WindowRef GetWindowRefFromNativeWindow(CGSWindowID wid) DEPRECATED_IN_MAC_OS_X_VERSION_10_5_AND_LATER;
-
-CG_EXTERN_C_END
